src/styles/style.css

```diff
@@ -38,78 +39,88 @@ body {
 
 ul {
   list-style-position: inside;
-  margin-top: 0.65rem;
+  margin-top: var(--gcds-spacing-300);
 }
 
 ul li {
-  margin-top: 0.433rem;
+  margin-top: var(--gcds-spacing-200);
 }
 
 /*
  * TYPOGRAPHY
  */
 
+h1,
+h2,
+h3,
+h4 {
+  font-family: var(--gcds-font-families-heading), sans-serif;
+}
+
+h5,
+h6 {
```

Comment:
I think the extra spacing is coming from the browser default padding and margins since we didn't have any reset styles. I added some now, would you mind checking again?